packages/investor-foxy/src/api/api.ts

```diff
@@ -154,71 +167,56 @@ export class FoxyApi {
   }
 
   checksumAddress(address: string): string {
-    return this.web3.utils.toChecksumAddress(address)
+    // ethers always returns checksum addresses from getAddress() calls
+    return ethers.utils.getAddress(address)
   }
 
   private verifyAddresses(addresses: string[]) {
     try {
       addresses.forEach(address => {
         this.checksumAddress(address)
       })
-    } catch (err) {
-      throw new Error(`Verify Address: ${err}`)
+    } catch (e) {
```

Comment:
haha i have been updating `e` -> `err`, we should decide/add lint rule if that exists